tests/emukit/quadrature/test_quadrature_kernels.py

```diff
@@ -366,7 +416,7 @@ def test_qkernel_qKq(kernel_embedding, interval):
             ),
         ),
         (
-            lebesgue_embeddings_test_list[2],
+            lebesgue_embeddings_test_list[3],
```

Comment:
Maybe these lists could be turned into dictionaries, so that such changes won't be necessary? `lebesgue_embeddings_test["qmatern32"]` also reads a bit better

Reply:
good idea! done.